.github/workflows/sync-docs.yaml

```
@@ -0,0 +1,72 @@
+name: Sync docs to docs site repo
+
+on:
+  pull_request:
+    branches:
+      - main
+    types:
+      - closed
```

```suggestion
  push:
    branches:
      - main
```

We might skip this `sync` during pushes to `main` or even as part of the [publish](https://github.com/slackapi/slack-github-action/blob/main/.github/workflows/publish.yml) workflow?

At the moment this might be skipped on releases, but TBH the `workflow_dispatch` is a life saver for this and other odd cases 🙏